src/components/ebay-window-notice/index.marko

```diff
@@ -0,0 +1,9 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+$ var status = "confirmation";
+<ebay-notice-base
+    class=[input.fillWindow && `window-notice--fill`, input.class]
```

Comment:
Just a heads up that I changed this modifier from `fill` to `screen` in v11 branch.